Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1824,6 +1824,7 @@ a_Player:OpenWindow(Window);
 				GetHealth = { Return = "number" },
 				AddEntityEffect = { Params = "EffectType, {{cEntityEffect}}", Return = "", Notes = "Applies an entity effect" },
 				RemoveEntityEffect = { Params = "EffectType", Return = "", Notes = "Removes a currently applied entity effect" },
+				HasEntityEffect = { Return = "true, if the entity effect is currently applied", Params = "EffectType", Notes = "Checks, if an entity effect is currently applied" }
```

Comment:
Could you replace `EffectType` with `{{cEntityEffect|EffectType}}`, and do the same for the two lines above as well?